tasks/mountpoint_present.yml

```diff
@@ -33,13 +32,25 @@
     mode: 0644
 
 - name: Check if OIOFS is already present in container
-  command: 'openio reference show {{ mountpoint.container | default(oiofs_mountpoint_default_container) }} --oio-account {{ mountpoint.account | default(oiofs_mountpoint_default_account) }} --oio-ns {{ mountpoint.namespace | default(oiofs_mountpoint_default_namespace) }} -c meta.oiofs_chunk_size'
+  command: "openio reference show {{ mountpoint.container | default(oiofs_mountpoint_default_container) }} \
+     --oio-account {{ mountpoint.account | default(oiofs_mountpoint_default_account) }} \
+     --oio-ns {{ mountpoint.namespace | default(oiofs_mountpoint_default_namespace) }} \
+     -c meta.oiofs_chunk_size"
   register: oiofs_mkfsed
   changed_when: false
-  ignore_errors: true
+  failed_when: false
 
 - name: "oiofs: Create the filesystem inside container"
-  shell: "mkfs.oiofs -C {{ openio_oiofs_conf_directory }}/{{ mountpoint_id }}.json -v {% if mountpoint.force_mkfs | default(oiofs_mountpoint_default_force_mkfs) %} -f {% endif %} -u {{ mountpoint.user | default(oiofs_mountpoint_default_user) }} -g {{ mountpoint.group | default(oiofs_mountpoint_default_group) }} -m {{ mountpoint.mode | default(oiofs_mountpoint_default_mode) }} -c {{ mountpoint.chunk_size | default(oiofs_mountpoint_default_chunk_size) }} -i  {{ mountpoint.inode_by_container | default(oiofs_mountpoint_default_inode_by_container) }} {{ mountpoint.namespace | default(oiofs_mountpoint_default_namespace) }}/{{ mountpoint.account | default(oiofs_mountpoint_default_account) }}/{{ mountpoint.container | default(oiofs_mountpoint_default_container) }}"
+  shell: "mkfs.oiofs -C {{ openio_oiofs_conf_directory }}/{{ mountpoint_id }}.json \
+    -v {% if mountpoint.force_mkfs | default(oiofs_mountpoint_default_force_mkfs) %} -f {% endif %} \
+    -u {{ mountpoint.user | default(oiofs_mountpoint_default_user) }} \
+    -g {{ mountpoint.group | default(oiofs_mountpoint_default_group) }} \
+    -m {{ mountpoint.mode | default(oiofs_mountpoint_default_mode) }} \
+    -c {{ mountpoint.chunk_size | default(oiofs_mountpoint_default_chunk_size) }} \
+    -i  {{ mountpoint.inode_by_container | default(oiofs_mountpoint_default_inode_by_container) }} \
+    {{ mountpoint.namespace | default(oiofs_mountpoint_default_namespace) }}/\
+    {{ mountpoint.account | default(oiofs_mountpoint_default_account) }}/\
+    {{ mountpoint.container | default(oiofs_mountpoint_default_container) }}"
```

Comment:
Isn't this adding unwanted whitespace in the "$NS/$ACCOUNT/$CONTAINER" string ?

Reply:
no